ckanext/validation/templates/validation/validation_read.html

```diff
@@ -1,51 +1,58 @@
+{% endblock breadcrumb_content %}
+
+{% block pre_primary %}  
+  {% set back_url = h.url_for(dataset.type ~ '_resource.edit', id=dataset.id, resource_id=resource.id) %}
+  {% snippet "package/snippets/step_indicator.html", step=2, back_url=back_url %}
+{% endblock pre_primary %}
+
+{% block form_title %}
+  {{ h.resource_display_name(resource) | truncate(50) }}
+  {{ h.get_validation_badge(resource)|safe }}
+{% endblock form_title %}
+
+{# Temporary until edit_resource_base.html is modified #}
+{% block page_header %}{% endblock page_header%}
+
+{% block primary_content_inner %}
+  <div class="validation-details">          
+      <div>{{ _('Validation timestamp') }}: {{ h.render_datetime(resource.validation_timestamp, with_hours=True) }}</div>
+      {% if validation.report %}
+      <div>{{ _('Duration') }}: {{ h.validation_dict(validation.report)["stats"]["seconds"] }}s</div>
+      {% endif %}
+  </div>
+  {% if validation.report %}
+      <div class="ontario-alert ontario-alert--success">
+        <div class="ontario-alert__header">
+            <div class="ontario-alert__header-icon">
+                <svg class="ontario-icon" alt="" aria-hidden="true" focusable="false" sol:category="primary" viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet">
+                    <use href="#ontario-icon-alert-success"></use>
+                </svg>
+            </div>
+            <h2 class="ontario-alert__header-title ontario-h4">Data validated successfully</h2>
         </div>
-
-        {% if validation.report %}
-            <div id="report" {% if h.bootstrap_version() == '2' %}class="bs2"{% endif %} data-module="validation-report" data-module-report="{{ validation.report }}"></div>
-        {% endif %}
-
-      </div>
-    </section>
-{% if h.use_webassets() %}
-    {% snippet 'validation/snippets/validation_asset.html', name='ckanext-validation/report-css' %}
-    {% snippet 'validation/snippets/validation_asset.html', name='ckanext-validation/report-js' %}
-{% else %}
-    {% snippet 'validation/snippets/validation_resource.html', name='ckanext-validation/report' %}
-{% endif %}
-
-{% endblock %}
-
-
-{% block primary %}{% endblock %}
-
-{% block secondary %}{% endblock %}
+    </div> 
+  {% endif %}
+
+  {% set package_id = resource.get('package_id') %}
+  {% set resource_id = resource.get('id') %}
+  {% set action = h.url_for('datastore.dictionary',
+            id=package_id,
+            resource_id=resource.id) %}
+  
+  <form method="get" action="{{ action }}" >
+    <button class="ontario-button ontario-button--primary" name="save" type="submit">Continue</button>
+  </form>
+
+  {% if h.use_webassets() %}
+      {% snippet 'validation/snippets/validation_asset.html', name='ckanext-validation/report-css' %}
+      {% snippet 'validation/snippets/validation_asset.html', name='ckanext-validation/report-js' %}
+  {% else %}
+      {% snippet 'validation/snippets/validation_resource.html', name='ckanext-validation/report' %}
+  {% endif %}
```

Comment:
This section is to display the validation report but I don't see the actual report on the step 2 page, do we need this section?

Reply:
We aren't displaying the validation report on this page and it doesn't show on this page. Let's leave this code in for now.

Reply:
What is the reasoning behind leaving this code in?

Reply:
To cover for situations where we will want the report displayed. But we can take it out here and add it back if we need it later.